### ISSUE TITLE
NTR: CT or MRI: sella turcica empty or pituitary absent, underdeveloped, or dysplastic (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8886]
New term request CT or MRI: sella turcica empty or pituitary absent, underdeveloped, or dysplastic (segal_060912205136):   UMLS: 4478860

**Comment by pnrobinson** [comment]
@MickeySegal
this term seems to encompass a variety of findings, see [Abnormal sella turcica morphology HP:0002679](https://hpo.jax.org/app/browse/term/HP:0002679). I have also created one new term

Adding new term: 
- ID: HP:6000483
- label: Empty sella turcica
- parents: [Abnormal sella turcica morphology (HP:0002679)](https://hpo.jax.org/app/browse/term/HP:0002679)
- pmid: PMID:22875743
- def: Empty sella or arachnoidocele has been defined as the herniation of the subarachnoid space within the sella turcica, associated with elongated pituitary stalk and flattening of the pituitary gland.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8886
- synonyms: Arachnoidocele(NONE)
